CHANGELOG/CHANGELOG-1.15.md

```diff
@@ -1,5 +1,22 @@
 # Changelog 1.15
 
+## 1.15.1 - 2024-03-18
+
+### Changed
+
+- Stop using `projects.registry.vmware.com` for user-facing images. ([#6073](https://github.com/antrea-io/antrea/pull/6073), [@antoninbas])
+- Persist TLS certificate and key of antrea-controller and periodically sync the CA cert to improve robustness. ([#5955](https://github.com/antrea-io/antrea/pull/5955), [@tnqn])
+- Update maximum number of buckets to 700 in OVS group add/insert_bucket message. ([#5942](https://github.com/antrea-io/antrea/pull/5942), [@hongliangl])
```

Comment:
Not sure if we need this in release log or not, @tnqn @antoninbas any comment?

Reply:
We can, but it's more of a fix IMO since the previous limit (800) was no longer valid and it could have lead to errors or inconsistencies

Reply:
Moved to fixed.